### DISCUSSION TITLE
Online maps cached tile, more tiles download

### DISCUSSION THREAD

**Discussion by drazde** [osmandapp/OsmAnd#16618] [Ideas]
Hi, 
I it possible download more tiles for region already have on the device?

I explain my trouble, I often go on mountain path and I found very useful have _Waymarkedtrails Hiking_ to see the correct information of a path or route (number and symbol).
Many times I'm not online when I use this overlap informations so I can't download and see it.

Yes I should see the map before i start hiking, but it would be helpful have more tiles already downloaded. 

Thanks

**Comment by mikehgentry** [comment]
If you have a raster map selected as the main map source (possibly as an overlay / underlay as well, don't know) you can long-press somewhere on it, then go to Actions -> Download map, and it'll let you draw a box to download more tiles.

[edit: Having tested it, it only works if it's the main map layer, at least until 4.4 (#16362), but you can easily make a profile like that specifically for downloading tiles]

You can bodge doing it along a route at a specific zoom level using simulated navigation (possibly you need to activate the development plugin, I can't remember), but it takes a while. I put a request in to make a nicer interface for it here #10181

**Reply by drazde** [reply]
> [edit: Having tested it, it only works if it's the main map layer, at least until 4.4 (#16362), but you can easily make a profile like that specifically for downloading tiles]

Ok thanks now I can do it!
